src/main/javascript/src/views/participation/manual-participation/ParticipationTypeManualSelection.vue

```diff
@@ -9,84 +9,171 @@
-        <template v-for="(item, index) in ['1', '2', '3', '4']">
-          <v-list-item :key="item">
-            <v-list-item-action>
-              <v-checkbox :input-value="active" color="primary"></v-checkbox>
-            </v-list-item-action>
-
-            <v-list-item-content>
-              <v-list-item-title>{{ item }}</v-list-item-title>
-            </v-list-item-content>
-          </v-list-item>
-          <v-divider v-if="index !== 3" class="mx-4" :key="item" />
-        </template>
-      </v-list-item-group>
-    </v-list>
+      <v-expansion-panel>
+        <v-expansion-panel-header>
+          Unassigned ({{ unassigned.length }})
+        </v-expansion-panel-header>
+        <v-expansion-panel-content>
+          <ListParticipants
+            :listOfParticipants="unassigned"
+            :moveToOptions="moveToOptions"
+            :moveToHandler="moveToHandler"
+            selectedOption="2"
+          />
+        </v-expansion-panel-content>
+      </v-expansion-panel>
+    </v-expansion-panels>
 
     <div class="mt-5">
-      <v-btn
-        elevation="0"
-        color="primary"
-        :to="{ name: 'ParticipationDistribution' }"
+      <v-btn elevation="0" color="primary" @click="submitParticipants()"
         >Continue
       </v-btn>
     </div>
   </div>
 </template>
 
 <script>
+import ListParticipants from "../../../components/ListParticipants.vue";
+import { mapActions, mapGetters } from "vuex";
+import store from "@/store";
+
 export default {
   name: "ParticipationTypeManualSelection",
   props: ["experiment"],
+  components: {
+    ListParticipants,
+  },
+  data() {
+    return {
+      moveToOptions: ["Participating", "Not Participating", "Unassigned"],
+    };
+  },
+  computed: {
+    ...mapGetters({
+      participants: "participants/participants",
+    }),
+
+    experimentId() {
+      return this.experiment.experimentId;
+    },
+
+    participating() {
+      return this.groupParticipants(true);
+    },
+
+    notParticipating() {
+      return this.groupParticipants(false);
+    },
+
+    unassigned() {
+      return this.groupParticipants(null);
+    },
+  },
+  methods: {
+    ...mapActions({
+      fetchParticipants: "participants/fetchParticipants",
+      updateParticipantsGroup: "participants/updateParticipantsGroup",
+      updateParticipants: "participants/updateParticipants",
+    }),
+
+    groupParticipants(value) {
+      return this.participants.filter(
+        (participant) => participant.consent === value
+      );
+    },
+
+    getParticipantIds(participants) {
+      return participants.map((participant) => participant.user.userId);
+    },
+
+    updateParticipantConsent(selectedIds, value) {
+      return this.participants.map((participant) => {
+        if (selectedIds.includes(participant.user.userId)) {
+          participant.consent = value;
+        }
+        return participant;
+      });
+    },
+
+    moveToHandler(option, tempSelected) {
+      const _this = this;
```

Comment:
In some instances, it is necessary to bind `_this`, but you shouldn't need to bind this scope to `_this` in this context. You can remove this line and change the following references of `_this` to `this`.

Reply:
Understood. Updating.